psi4/driver/json_wrapper.py

```diff
@@ -178,6 +178,14 @@ def _clean_psi_environ(do_clean):
         psi4.core.clean_options()
         psi4.core.clean()
 
+def _read_output(outfile):
+    try:
+        with open(outfile, 'r') as f:
+            output = f.read()
+
+        return output
```

Comment:
should be w/i context manager? or `return f.read()` since probable error in the `open` line

Reply:
Unclear what happens in this case if closing the file fails? This seems safer.

Reply:
try/except/finally, then, to minimize lines in `try`? approving regardless

Reply:
We could, but why try to minimize lines in try here? We want something that cannot fail.

Reply:
If anything, I guess `try/except/else` would work (`finally` gets executed even on exceptions, which I guess we don't want), but I don't see a problem with this structure.